cpp/cmake_modules/SetupCxxFlags.cmake

```diff
@@ -642,6 +642,9 @@ elseif("${CMAKE_BUILD_TYPE}" STREQUAL "PROFILE_GEN")
 elseif("${CMAKE_BUILD_TYPE}" STREQUAL "PROFILE_BUILD")
   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${C_FLAGS_PROFILE_BUILD}")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CXX_FLAGS_PROFILE_BUILD}")
+elseif("${CMAKE_BUILD_TYPE}" STREQUAL "MINSIZEREL")
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${C_FLAGS_MINSIZEREL}")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CXX_FLAGS_MINSIZEREL}")
```

Comment:
This should probably reuse `C[XX]_FLAGS_RELEASE` actually (note that `C_FLAGS_MINSIZEREL` is not defined in this file).